cdap-docs/reference-manual/source/release-notes.rst

```diff
@@ -23,6 +23,15 @@ Cask Data Application Platform Release Notes
    :backlinks: none
    :depth: 2
 
+`Release 3.3.2 <http://docs.cask.co/cdap/3.3.2/index.html>`__
+=============================================================
+
+Bug Fixes
+---------
+- `CDAP-5019 <https://issues.cask.co/browse/CDAP-5019>`__ - Fixed an issue with
+  launching Spark jobs on Cloudera Manager clusters when using Spark 1.4 or greater.
```

Comment:
Do we call it "Cloudera Manager" clusters or "Clusters created with Cloudera Manager"?